### DISCUSSION TITLE
How to Install latest version of protobuf using vcpkg ?

### DISCUSSION THREAD

**Discussion by nharis7** [microsoft/vcpkg#35218] [Q&A]
I have installed vcpkg on my windows machine. I am installaing protobuf but it installs 3.21.12 version and I could not find a way to install the latest version. I have read manifest mode documentation, it creates vcpkg.json file and can add dependency but does not provide any information on how to install latest version of some package.

**Comment by autoantwort** [comment]
`3.21.12` is the latest version available in vcpkg. There is https://github.com/microsoft/vcpkg/pull/32602 to update to `23.4`, you could try this PR.

**Reply by nharis7** [reply]
So, I cannot install v25.0 using vcpkg ?

**Reply by autoantwort** [reply]
Not out of the box. You can change the port so that it uses version `v25.0`
